src/services/help-modal-service/help-texts/usertask-usage.ts

```diff
@@ -0,0 +1,50 @@
+import {HelpText} from '../../../contracts/index';
+
+export const UserTaskUsage: HelpText = {
+  title: 'How to use user tasks',
+  message: `The data of previous tasks can be accessed through the token.
```

Comment:
Ich würde hier auch den einleitenden Abschnitt zur token Erklärung mit einschließen:

```suggestion
  message: `When a process is executed, it progresses through the different elements of the diagram and captures states along the way.
  This progression and state accumulation can be visualized as a token moving along the execution's path in the diagram.
  
  The data of previous tasks can be accessed through the token.
```